recipe/0001-set-version.patch

```diff
@@ -0,0 +1,18 @@
+For some reason, it's not enough to source this file and then override the variable.
+
+We have to patch it. It must be getting sourced again by internal Quarto code
+at some point.
+
+diff --git a/configuration b/configuration
+index 0b6bef3..557ef9c 100644
+--- a/configuration
++++ b/configuration
+@@ -44,7 +44,7 @@ export ALGOLIA_SEARCH_INSIGHTS_JS=2.0.3
+ 
+ 
+ # Quarto Info Version
+-export QUARTO_VERSION=1.3
++export QUARTO_VERSION=${PKG_VERSION}
```

Comment:
You can't use variable expansions in this file. This file gets used for both windows and linux/mac, with a regex pattern to translate. The current broken windows build is because the version is literally the string `${PKG_VERSION}` on windows. That's why the QUARTO_VERSION was being set in bld.bat and build.sh (with .bat variable expansion on the windows side, of course.)

I recommend removing this patch and setting things in those conda recipe build scripts instead.

Reply:
When we were doing to `QUARTO_VERSION` setting in the build scripts, I was seeing the version as `1.3.1`: https://github.com/conda-forge/quarto-feedstock/pull/7#issuecomment-1539313171

Any ideas how to get past that?